.github/workflows/bump-versions.yaml

```diff
@@ -0,0 +1,112 @@
+name: Bump chart and/or app versions
+permissions:
+  contents: write
+  pull-requests: write
+on:
+  workflow_call:
+    inputs:
+      bypass-dev-protections:
+        description: >
+          Use custom user to override branch protections when
+          pushing directly to dev
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      PUSH_CHARTS_TOKEN:
+        required: false
+
+jobs:
+  # Increment the APP and CHART versions based on commit message starting with Update-versions 
+  bump-versions:
+    name: Get supported Go versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Calculate desired version changes
+        id: get-new-versions
+        run: |
+          . version.env
+          outputs=""
+          for commit in ${{ github.event.push.commits }}; do
+            if [[ "${commit.message}" == *"Update-versions "* ]]; then
```

Comment:
@austinorth opinion on this: Right now it has to match "Update-versions " with the space and case sensitive. Do you think this should be loosened? Things to consider: it has default patch chart version behavior, so if someone adds this in their commit accidentally and doesn't expect a patch it might bump their versions on top of them having done it manually (presumably)
But also consider these are commit messages written by humans so they might want to use this and struggle with the caps and the trailing space. Maybe the - is enough to avoid accidental bumps and the rest should be no trailing space needed and case insensitive? 
Open to opinions here

Reply:
Honestly, I would forward this question to @MikeRawding who originally requested this feature lol. I'm not sure the level of flexibility/rigidity teams are expecting with this!

Reply:
I think it's fine to be case sensitive.  I might suggest it just be all lower case since this sort of feels like a "command".